hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -202,9 +202,14 @@ public KafkaOffsetGen(TypedProperties props) {
     // Come up with final set of OffsetRanges to read (account for new partitions, limit number of events)
     long maxEventsToReadFromKafka = props.getLong(Config.MAX_EVENTS_FROM_KAFKA_SOURCE_PROP,
         Config.maxEventsFromKafkaSource);
-    maxEventsToReadFromKafka = (maxEventsToReadFromKafka == Long.MAX_VALUE || maxEventsToReadFromKafka == Integer.MAX_VALUE)
```

Comment:
Why do you remove this check statement? If the user sets the value of `Config.MAX_EVENTS_FROM_KAFKA_SOURCE_PROP` to `Long.MAX_VALUE`. How do you reset it to `Config.maxEventsFromKafkaSource`(5000000)?

Reply:
> 
> 
> Why do you remove this check statement? If the user sets the value of `Config.MAX_EVENTS_FROM_KAFKA_SOURCE_PROP` to `Long.MAX_VALUE`. How do you reset it to `Config.maxEventsFromKafkaSource`(5000000)?

detailed discuss is here: https://issues.apache.org/jira/projects/HUDI/issues/HUDI-340?filter=allissues
this check can not achieve its goal to set a limit to max events read from kafka in one batch, and this goal is not absolute，it is allowable to set a bigger num than `Config.maxEventsFromKafkaSource` as discussed above.

Reply:
Got it. Sg.